plumbing/api.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 
 	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	"gx/ipfs/QmVRxA4J3UPQpw74dLrQ6NJkfysCA1H4GU28gVpXQt9zMU/go-libp2p-pubsub"
```

Comment:
we could and maybe should take this a step further by defining a local pubsub package at the top level and an interface in that package that all consumers in gfc uses and then have a constructor in that package that returns an instance of this pubsub. that way if we change the implementation out it is only a small change. could do as a follow on or not, up to you.